app/src/main/java/org/mozilla/focus/menu/browser/BrowserMenuAdapter.kt

```diff
@@ -69,48 +69,48 @@ class BrowserMenuAdapter(
         items.add(MenuItem.BlockingSwitch)
 
         if (customTabConfig == null || customTabConfig.showShareMenuItem) {
-            items.add(MenuItem.Default(R.id.share, resources.getString(R.string.menu_share)))
+            items.add(MenuItem.Default(R.id.share, resources.getString(R.string.menu_share), R.drawable.ic_share_menu_item))
         }
 
-        items.add(MenuItem.Default(R.id.add_to_homescreen, resources.getString(R.string.menu_add_to_home_screen)))
-        items.add(MenuItem.Default(R.id.find_in_page, resources.getString(R.string.find_in_page)))
+        items.add(MenuItem.Default(R.id.add_to_homescreen, resources.getString(R.string.menu_add_to_home_screen), R.drawable.ic_add_to_home_menu_item))
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.